person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private :of_age?
+
+  def of_age?
+    return unless @age >= 18
```

Comment:
- [ ] you can remove the unless here because it might return `undefined` if it is false in this case, however we want this method to return either `true` or `false`, which it will do if you write it like below:
```
def of_age?
    @age >= 18
end
```